source/blog/2017-02-14-announcing-hanami-100beta1.html.markdown

```diff
@@ -8,30 +8,30 @@ excerpt: >
   Feature freeze, project logger, automatic logging of requests, SQL queries, and migrations. Minor bug fixes.
 ---
 
-This `v1.0.0.beta1` release introduces Hanami [feature freeze](https://en.wikipedia.org/wiki/Freeze_(software_engineering)) for 1.0, alongside with a couple of features, and a few bug fixes.
+This `v1.0.0.beta1` release marks Hanami's [feature freeze](https://en.wikipedia.org/wiki/Freeze_(software_engineering)) for 1.0, along with a couple new features, and a few bug fixes.
 
-Since now on, **Hanami API's are stable and won't be changed until 2.0**.
+From now on, **Hanami API's are stable and won't be changed until 2.0**.
```

Comment:
`API's` is correct :) https://afterdeadline.blogs.nytimes.com/2010/04/13/faqs-on-style/?_r=0